controller/src/beerocks/bml/bml.cpp

```diff
@@ -181,16 +182,54 @@ int bml_event_register_cb(BML_CTX ctx, BML_EVENT_CB cb)
     return pBML->register_event_cb(cb);
 }
 
-int bml_set_wifi_credentials(BML_CTX ctx, const char *ssid, const char *pass, int sec, int vap_id,
-                             int force)
+int bml_set_wifi_credentials(BML_CTX ctx, const char *al_mac, const char *ssid,
+                             const char *network_key, const char *operating_class,
```

Comment:
(nitpick)
```suggestion
                             const char *network_key, const char *bands,
```
The string is not an operating class, it's a band specification.